edbdeploy/data/tpaexec/hooks/post-deploy.yml

```diff
@@ -1,5 +1,51 @@
 ---
 
+- name: "Get the information of {{ pg_systemd_alias }} service"
+  stat:
+    path: "/etc/systemd/system/edb-as-{{ postgres_version }}.service"
+  when:
+    - "'bdr' in role"
+    - "'primary' in role"
+    - pg_systemd_alias is defined
+    - pg_systemd_service_path is defined
+  register: pg_service_stat
```

Comment:
This variable does not look to be used further.

Reply:
fixed